frontend/src/pages/PlannerWeightsPage.tsx

```diff
@@ -56,79 +48,33 @@ export const PlannerWeightsPage = () => {
     if (!currentRoadmap) dispatch(roadmapsActions.getRoadmaps());
   }, [dispatch, currentRoadmap]);
 
-  const resetWeights = () => {
-    dispatch(roadmapsActions.clearPlannerCustomerWeights());
-  };
-
-  useEffect(() => {
-    setNoChanges(
-      !customerWeights.some(({ customerId, weight }) => {
-        const saved = customers?.find(({ id }) => id === customerId)?.weight;
-        return weight !== saved;
-      }),
+  const handleSliderChange = async (customerId: number, weight: number) => {
+    setLocalCustomers(
+      localCustomers!.map((customer) =>
```

Comment:
It's better to provide update function to set state, when the previous state needs to be referenced,
this way e.g. batched updates work reliably
```suggestion
    setLocalCustomers((previous) =>
      previous!.map((customer) =>
```